### ISSUE TITLE
[Suggestion] Security page

### ISSUE THREAD

**Issue by bykidi** [Wruczek/ts-website#39]
1. Custom SQ account
2. Needed privileges for it
etc etc

**Comment by Wruczek** [comment]
Yep! It is in fact planned to made the new ts-website 2.0 installer notice about safe practices and securing private folders. But wiki page will be useful too.
